src/debug-macros.js

```diff
@@ -45,9 +45,15 @@ function getFlags(app, isDevelopingAddon) {
   // >= 3.15.0.beta
   features = app.project.require('@ember-data/private-build-infra/src/features')(isProd);
 
-  features = Object.assign({ CUSTOM_MODEL_CLASS: false }, features);
+  features = Object.assign(
+    {
+      CUSTOM_MODEL_CLASS: false,
+      PROXY_MODEL_CLASS: false,
+    },
+    features
+  );
```

Comment:
this seems odd; how do we enable this at build time?  looks like we're pretending it's an Ember Data feature?